tests/contrib/affiliations/conftest.py

```diff
@@ -33,6 +33,33 @@ def affiliation_full_data():
     }
 
 
+@pytest.fixture(scope="function")
+def affiliation_openaire_data():
```

Comment:
minor: `openaire` or `cordis` data?

Reply:
* OpenAIRE provides extra info about affiliations, by providing the mapping between ROR and PIC for some organizations.
* CORDIS provides extra info about awards, by providing subjects and participating organizations.